spec/features/todo_list_item_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature "Todo list item", :type => :feature do
+
+  scenario "create to-do item" do
+    user = sign_in("tester@example.tld", "test-password")
+    create(:list, user: user, title: "to-do list")
+
+    visit lists_path
+    click_link "to-do list"
+
+    list = List.last
+    expect(current_path).to eq(list_path(list))
+
+    expect(page).to have_content("to-do list")
+    click_link "Create new to-do"
+
+    expect(page).to have_content("Create a new to-do item")
+    fill_in "Name", with: "purchase groceries"
+    click_button "Create new to-do"
+
+    expect(current_path).to eq(list_path(list))
+    expect(page).to have_content("Your to-do was created.")
+    expect(page).to have_content("purchase groceries")
```

Comment:
Ah yes, you did mention previsouly, something along the lines of using real worldish data/content. Ta